src/bench/checkqueue.cpp

```diff
@@ -26,6 +24,10 @@ static const unsigned int QUEUE_BATCH_SIZE = 128;
 // and there is a little bit of work done between calls to Add.
 static void CCheckQueueSpeedPrevectorJob(benchmark::Bench& bench)
 {
+    if (GetNumCores() < 2) {
+        return;
+    }
```

Comment:
I think this warrants a comment explaining "why?".

Also, maybe don't drop the `MIN_CORES` constant and use it here instead of the magic number "2"?